interview-questions.md

```diff
@@ -7,44 +7,55 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. As a developer, I am creating a Rails application with a model called Cohort that has_many students, but OOPS! I forgot to add the foreign key. How can I fix this mistake? What is the name of the foreign key? Would the foreign key be on the Cohort model or the Student model?
 
-  Your answer:
+  Your answer: This invloves associations in the Cohort Model you would have a has_many relationship. A cohort has many students.
 
-  Researched answer:
+  Researched answer: An association is a connection between two Active Record models. Rails supports 6 types of associations.
 
 
 
 2. Which RESTful routes must always be passed params? Why?
 
-  Your answer:
+  Your answer: GET,PATCH,DELETE will require a params. In the form of :id.  
```

Comment:
Those are definitely the http verbs associated with the RESTful route, but what are the action names?  Also, one line answers in an interview will not make you stand out.  Oftentimes it can come off as uninterested or unmotivated.  I would highly recommend working on elaborating further on the concepts.